Attribute Analysis.ipynb

```diff
@@ -971,6 +974,122 @@
     "interact(cooccurrence_counts_mi, topn=widgets.IntSlider(min=1, max=30, step=1, value=10));"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# att_scn Metric: Scenes\n",
+    "<a id=\"att_scn\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "<a id=\"att_scn_setup\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hide_toggle(for_next=True, toggle_text='Show/hide att_scn code')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stats_info = pickle.load(open('results/{}/att_scn.pkl'.format(folder_name), 'rb'))\n",
+    "scenes_per = stats_info['scenes_per']\n",
+    "print(len(scenes_per[0]))\n",
+    "info = pickle.load(open('util_files/places_scene_info.pkl', 'rb'))\n",
+    "idx_to_scene = info['idx_to_scene']\n",
+    "idx_to_scenegroup = info['idx_to_scenegroup']\n",
+    "sceneidx_to_scenegroupidx = info['sceneidx_to_scenegroupidx']\n",
+    "\n",
+    "xaxis = [idx_to_scenegroup[i] for i in range(len(idx_to_scenegroup))]\n",
+    "xaxis = ['\\n'.join(textwrap.wrap(chunk, width=30)) for chunk in xaxis]\n",
+    "barWidth = .6\n",
```

Comment:
can this be set to a constant or is it dependent on how many attributes there are?